src/js/binaryen.js-post.js

```diff
@@ -689,30 +689,30 @@ function wrapModule(module, self = {}) {
   }
 
   self['memory'] = {
-    'size'() {
-      return Module['_BinaryenMemorySize'](module);
+    'size'(name) {
+      return Module['_BinaryenMemorySize'](module, strToStack(name));
```

Comment:
I see we have existing bugs here, but every `strToStack` call needs to be wrapped in a `preserveStack` call to avoid permanently leaking stack space.